subprojects/enterprise-operations/src/main/java/org/gradle/operations/execution/CachingDisabledReasonCategory.java

```diff
@@ -14,13 +14,12 @@
  * limitations under the License.
  */
 
-package org.gradle.api.internal.tasks;
+package org.gradle.operations.execution;
```

Comment:
:x: We should converge on all operations living under `org.gradle.enterprise.operations`.

The classes in the subproject are not currently under that package, but the reason for that is because we wanted to move them from some pre-existing location without changing the package as a first step. We should figure out how we will be able to migrate them under `org.gradle.enterprise.operations` eventually.

Regardless of the above, new types like this one should start moving towards that future.

Reply:
We decided that the package for the operations is `org.gradle.operations`, not `org.gradle.enterprise.operations`. See https://docs.google.com/document/d/1Ud1bQQcKOLmA6JpMXa6ayc-y_4S2miSnJAGdv7PCCWQ/edit#heading=h.ux5bj830etu8.

Reply:
Hm, and why? The linked GE discovery doc doesn't even mention that option, and all I see is at some point you deleted `enterprise` without explanation.

Reply:
The main reasons to end up with a package that doesn't have `enterprise` in its name were that
- It seems like `enterprise` doesn't add much context to the package.
- The operations in there might be used for local build scans as well, and `enterprise` didn't seem fitting for this.

Reply:
That makes sense, though then the subproject name should also not include `enterprise`, no?

Reply:
No disagreement from my side. We were going to do this when we migrate the project to a platform.